docs/ops/image/Interpolate_4.md

```diff
@@ -16,6 +16,16 @@
   * **Default value**: none
   * **Required**: *yes*
 
+* *shape_calculation_mode*
+
+  * **Description**: specifies which of inputs `target_spatial_shape` or `scales` is used to calculate an output shape
```

Comment:
Is it really `target_spatial_shape`? As far as I can see the C++ code it should be called `sizes`.

Reply:
Fixed.